server/app/services/question/types/MultiOptionQuestionDefinition.java

```diff
@@ -25,14 +25,17 @@ public final class MultiOptionQuestionDefinition extends QuestionDefinition {
 
   public MultiOptionQuestionDefinition(MultiOptionQuestionDefinitionConfig config) {
     super(
-        config.id(),
-        config.name(),
-        config.enumeratorId(),
-        config.description(),
-        config.questionText(),
-        config.questionHelpText(),
-        config.validationPredicates().orElse(MultiOptionValidationPredicates.create()),
-        config.lastModifiedTime());
+        QuestionDefinitionConfig.builder()
```

Comment:
question (blocking): Why doesn't this follow the same pattern as the other question types with just `super(config);`?

Reply:
It's because `MultiOptionQuestionDefinitionConfig` existed before this PR and is deeply embedded throughout the codebase. In hindsight I should have done this migration at once with `QuestionDefinitionConfig` alone, but for now this is a good state to be in until we migrate `MultiOptionQuestionDefinitionConfig` too. I added a TODO above the `MultiOptionQuestionDefinition` constructor.